travel/src/main/resources/db/migration/V3__dbscheme.sql

```diff
@@ -0,0 +1,84 @@
+CREATE TABLE IF NOT EXISTS `exchange_rates` (
+    `id` BIGINT NOT NULL AUTO_INCREMENT,
+    `code` VARCHAR(3) DEFAULT NULL,
+    `amount_in_rsd` DECIMAL(7,6) UNSIGNED DEFAULT NULL,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `UK_code` (`code`)
+    );
+
+CREATE TABLE IF NOT EXISTS `travel_expense_forms` (
+    `id` VARCHAR(36) NOT NULL,
+
+    `business_trip_end_date` TIMESTAMP NOT NULL,
+    `breakfasts_provided` TINYINT UNSIGNED DEFAULT 0,
+    `lunches_provided` TINYINT UNSIGNED DEFAULT 0,
+    `dinners_provided` TINYINT UNSIGNED DEFAULT 0,
+    `submission_date` TIMESTAMP NOT NULL,
+    PRIMARY KEY (`id`)
+    );
+
+CREATE TABLE IF NOT EXISTS `expenses` (
+    `id` VARCHAR(36) NOT NULL,
+    `claim` VARCHAR(255) NOT NULL,
+    `description` VARCHAR(255),
+    `amount` TINYINT UNSIGNED DEFAULT 1,
+    `currency_code` VARCHAR(3),
+    `tef_id` VARCHAR(36) NOT NULL,
+    PRIMARY KEY (`id`),
+    CONSTRAINT `FK_expense_tef_id_constraint` FOREIGN KEY (`tef_id`) REFERENCES `travel_expense_forms` (`id`)
+    );
+
+CREATE TABLE IF NOT EXISTS `travel_authorization_forms` (
+    `id` VARCHAR(36) NOT NULL,
+
+    `business_trip_end_date` TIMESTAMP NOT NULL,
+    `arrival_date` TIMESTAMP NOT NULL,
+    `accommodation` VARCHAR(255) NOT NULL,
+    `breakfasts` TINYINT UNSIGNED NOT NULL,
```

Comment:
maybe name this as breakfast_number?

Reply:
Thanks for the suggestion, I see how this might be incomplete information about what's stored in that column.
I will change it 😄